Gemfile

```diff
@@ -49,6 +51,8 @@ group :development do
 end
 
 group :test do
+  gem 'pry-byebug'
+  gem 'pry-rails'
```

Comment:
I used these gems in the test environment in order to debug specs. Of course, I can remove them, as I don't need them anymore, but I find them quite handy. We could get rid of of warning, if we use the syntax:

```ruby
group :development, :test do
  ...
end
```

Should I remove them or use the syntax?